assets/src/edit-story/elements/text/display.js

```diff
@@ -175,14 +175,17 @@ function TextDisplay({
   useTransformHandler(id, (transform) => {
     // Ref is set in case of high-light mode only, use the fgRef if that's missing.
     const target = ref?.current || fgRef.current;
-    const updatedFontSize = transform?.updates?.fontSize;
-    target.style.fontSize = updatedFontSize
-      ? `${dataToEditorY(updatedFontSize)}px`
-      : '';
-    const updatedMargin = transform?.updates?.marginOffset;
-    target.style.margin = updatedMargin
-      ? `${dataToEditorY(-updatedMargin) / 2}px 0`
-      : '';
+    if (target) {
```

Comment:
There's a conflicting PR resolving the same issue in a bit different way: https://github.com/google/web-stories-wp/pull/7147/files

In that PR it just returns if the `target` doesn't exist, not sure if it makes a difference. cc @barklund

Reply:
Oh, I didn't see that one! Or well, I saw it once but didn't realize it's the same thing.

Merging that one since it already went through QA